orttraining/orttraining/training_api/optimizer.h

```diff
@@ -64,8 +84,20 @@ struct Optimizer {
 
   Status Step();
 
+  /**
+   * @brief Get current optimizer state and store the CPU copy in optimizer_checkpoint_states.
+   *
+   * Be noted there are a copy underlying between devices (CPUtoCPU, CUDAtoCPU).
```

Comment:
Be noted there is an underlying copy between devices

Reply:
why is there a CPU to CPU copy?

Reply:
Removed as we talked offline.